docs/installation/prepare/jetson-nano-setup.md

```diff
@@ -66,6 +66,29 @@ If this command fails, try using `wget https://storage.googleapis.com/packages.v
 
 {{< readfile "/static/include/install/install-linux-aarch.md" >}}
 
+## Serial Communication Protocol Tips
+
+To change the pins that are in use for modes of serial communication, launch <file>jetson-io.py</file> with the following command:
+
+```sh { class="command-line" data-prompt="$"}
+sudo /opt/nvidia/jetson-io/jetson-io.py
+```
+
+In the interactive menu that opens, select **Configure Jetson 40 Pin Header** and **Configure header pins manually** to select and deselect pins to enable use.
+For a Jetson Orin Nano, reference the following:
+
+<!-- prettier-ignore -->
+| GPIO Header Pin | Viam Bus ID | `jetson-io.py` ID |
+| ---------------| ----------- | ----------------- |
+| 3, 5 | `7` | `i2c2` |
+| 27, 28 | `1` | `i2c8` |
+| 19, 21, 23, 24, 26 | `0` | `spi1` |
```

Comment:
There's a second SPI bus on pins 13, 16, 18, 22, and 37, though I haven't tried it out myself.

Reply:
```suggestion
| 19, 21, 23, 24, 26 | `0` | `spi1` |
| 13, 16, 18, 22, 37 |  | `spi3`|
```
@penguinland What would Viam bus ID be? Could add but don't want to leave that info out and I don't know

Reply:
It's `2`.

Reply:
Thank you!!